lib/halfnarp.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
```

Comment:
Shouldn't this be a sidekiq-task which updates the metadata when a new conferences / release is added?